### PR TITLE
Clarify primary key

### PR DESCRIPTION
**Pull request by maryamsulemani97**
Clarify primary key. Closing https://github.com/meilisearch/documentation/issues/934

### CODE REVIEW

**Comment by tpayet** [comment]
@alallema I am adding you as reviewer since you opened the associated issue :)

File: reference/api/documents.md [hunk 1]
```diff
@@ -108,7 +108,7 @@ If the provided index does not exist, it will be created.
 
 | Query Parameter | Description                                                                                     | Default Value |
 | --------------- | ----------------------------------------------------------------------------------------------- | :-----------: |
-| **primaryKey**  | The [primary key](/learn/core_concepts/documents.md#primary-key) of the documents _(optional)_ |     none      |
+| **primaryKey**  | The [primary key](/learn/core_concepts/documents.md#primary-key) of the index _(optional)_ |     none      |
```
Comment (alallema) [inline reference/api/documents.md:111]: It's better! Thanks

**Review by alallema — APPROVED** [review]

**Review by tpayet — APPROVED** [review]
Thank you @alallema :) 
@maryamsulemani97 you can merge 🚀

**Comment by maryamsulemani97** [comment]
bors merge

**Pull request merged**